packages/react-spinbutton/src/components/SpinButton/SpinButton.types.ts

```diff
@@ -1,20 +1,163 @@
-import type { ComponentProps, ComponentState, IntrinsicShorthandProps } from '@fluentui/react-utilities';
+import type * as React from 'react';
+import type {
+  ComponentProps,
+  ComponentState,
+  IntrinsicShorthandProps,
+  ObjectShorthandProps,
+} from '@fluentui/react-utilities';
+import { LabelProps } from '@fluentui/react-label';
 
 export type SpinButtonSlots = {
   // TODO Add slots here and to spinButtonShorthandProps in useSpinButton.ts
+  /**
+   * The root element of SpinButton is a container `<div>`.
+   * The root slot receives the `className` and `style` specified on the `<SpinButton>`.
+   * All other native props are applied to the primary slot: `input`.
+   */
   root: IntrinsicShorthandProps<'div'>;
+
+  /**
+   * Input that displays the current value and accepts direct input from the user.
+   * Displayed value is formatted.
+   *
+   * This is the primary slot.
+   */
+  input: IntrinsicShorthandProps<'input'>;
+
+  /**
+   * Renders the increment button.
+   */
+  incrementButton: IntrinsicShorthandProps<'button'>;
```

Comment:
I don't think that currently have a pattern with including HTML element into a slot name. Let's imagine that someday there will be a request to implement support for this slot to be an anchor element:

```tsx
<SpinButton incrementButton={{ as: 'a' }} /> 
```

In that context `incrementButton` looks a bit strange. I propose to use either `increment` or `incrementControl` if there is sense to be more verbose.

Reply:
I find `increment` (and `decrement`) confusing as I read them as verbs so I expect them to be linked to an action like an event handler. I agree that we should drop "Button" to allow for future expansions of elements. For now I'll change this to `incrementControl` but I'm open to other names.